examples/dreambooth/train_dreambooth.py

```diff
@@ -647,6 +647,13 @@ def load_model_hook(models, input_dir):
 
     if args.enable_xformers_memory_efficient_attention:
         if is_xformers_available():
+            import xformers
+
+            xformers_version = version.parse(xformers.__version__)
+            if xformers_version.micro == 16:
```

Comment:
Maybe check the other components too, so we don't fail in xFormers `0.1.16` ?

Reply:
Also let's try to use the same "style" consistently. We always use `package.version.parse(...) < ...` -> let's try to use this everywhere :-)

Reply:
Haha yes, that's a good point :grin: !

Reply:
done!